Lib/multiprocessing/resource_tracker.py

```diff
@@ -161,7 +161,7 @@ def unregister(self, name, rtype):
     def _send(self, cmd, name, rtype):
         self.ensure_running()
         msg = '{0}:{1}:{2}\n'.format(cmd, name, rtype).encode('ascii')
-        if len(name) > 512:
+        if len(msg) > 512:
             # posix guarantees that writes to a pipe of less than PIPE_BUF
             # bytes are atomic, and that PIPE_BUF >= 512
             raise ValueError('name too long')
```

Comment:
Shouldn't this also be updated? ie. `msg too long`?

Reply:
@DanielNoord 
Thanks for your comment!
The error message should also be updated. I will fix it.